src/components/GameOverShroud.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="game-over-shroud">
+
+    <div>
+      Game Over
+    </div>
+
+    <div>
+      Score: {{score}}
+    </div>
+
+    <div v-on:click="newGame" class="new-game-shroud-button">
+      Try Again
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
```

Comment:
The `vue-loader` in webpack uses this exported object to create a vue component.  That's how it works.